spyrmsd/parallel.py

```diff
@@ -0,0 +1,139 @@
+try:
+    from pebble import ProcessPool
+except ImportError:
+    errmsg = (
+        "Parallel execution of SPyRMSD (`prmsdwrapper`) requires `pebble`."
+        "Please install `pebble`: https://github.com/noxdafox/pebble"
+    )
+    raise ImportError(errmsg)
+
+import os
+import warnings
+from concurrent.futures import TimeoutError
+from functools import partial
+from typing import List, Optional, Union
+
+import numpy as np
+
+from spyrmsd import molecule
+from spyrmsd.rmsd import rmsdwrapper
+
+
+def prmsdwrapper(
+    molrefs: Union[molecule.Molecule, List[molecule.Molecule]],
+    mols: Union[molecule.Molecule, List[molecule.Molecule]],
+    symmetry: bool = True,
+    center: bool = False,
+    minimize: bool = False,
+    strip: bool = True,
+    cache: bool = True,
+    num_workers: Optional[int] = None,
+    timeout: Optional[float] = None,
+    chunksize: int = 1,
+) -> List[float]:
+    """
+    Compute RMSD between two molecules with a timeout.
+    Parameters
+    ----------
+    molrefs: Union[molecule.Molecule, List[molecule.Molecule]]
+        Reference molecule
+    mols: Union[molecule.Molecule, List[molecule.Molecule]]
+        Molecules to compare to reference molecule
+    symmetry: bool, optional
+        Symmetry-corrected RMSD (using graph isomorphism)
+    center: bool, optional
+        Center molecules at origin
+    minimize: bool, optional
+        Minimised RMSD (using the quaternion polynomial method)
+    strip: bool, optional
+        Strip hydrogen atoms
+    cache: bool, optional
+        Cache graph isomorphisms
+    num_workers: int
```

Comment:
```suggestion
    num_workers: int, optional
```
Was there a reason why not set it directly to `os.cpu_count()`?

Reply:
The original reasoning was to make it user-configurable, and when it wasn't specified, it defaulted to using the maximum amount of cores.